src/test/java/uk/co/sleonard/unison/utils/tests/StringUtilsTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.co.sleonard.unison.utils.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Vector;
+
+import org.junit.Test;
+
+import uk.co.sleonard.unison.utils.StringUtils;
+
+public class StringUtilsTest {
+
+	@Test
+	public void testConvertStringToList() {
+		
+		Vector<String> list = StringUtils.convertStringToList("string-to-list-test", "-");
+		
+		assertEquals(4, list.size());
+		
+	}
+	
+	@Test
+	public void testConvertStringToListIfNull() {
+		
+		final Vector<String> listEmpty = new Vector<String>();

```

Comment:
With Java 7 we can do = new Vector<>(); missing the type as it knows it already from the left side